package.json

```diff
@@ -67,7 +67,9 @@
   "peerDependencies": {
     "antd": "~3.0",
     "ol": "~4.0",
-    "react": "~16.0"
+    "react": "~16.0",
+    "ag-grid": "17.1.1",
```

Comment:
I'd suggest to use `~17` as peerDependencies shouldn't be to strict.

Reply:
Done.